ios/RCTTWVideoModule.m

```diff
@@ -598,28 +617,13 @@ -(TVIVideoBandwidthProfileOptions*)prepareBandwidthProfile:(NSDictionary *)bandw
     builder.roomName = roomName;
 
     if(encodingParameters[@"enableH264Codec"]){
-        if ([encodingParameters[@"enableH264Codec"] boolValue] == true) {
-            builder.preferredVideoCodecs = @[ [TVIH264Codec new] ];
-            NSLog(@"Preferring H264 Codec");
-        }
+      builder.preferredVideoCodecs = @[ [TVIH264Codec new] ];
     }
 
-    if (encodingParameters[@"audioBitrate"] && encodingParameters[@"videoBitrate"]) {
+    if(encodingParameters[@"audioBitrate"] || encodingParameters[@"videoBitrate"]){
```

Comment:
It seems like the upstream code is simpler and includes defaults if only one of the pair is provided (rather than logging an error).

I've been adopting a strategy of "if in doubt, use upstream code as it's means our fork is less diverged".

I'm more than happy to restore our implementation here if you prefer it?